src/actions/dashboard.js

```diff
@@ -100,14 +118,15 @@ export const fetchEvents = (date, eventView, force = false) => dispatch => {
     };
 
     // Fetch personal events only.
-    if (eventView === eventsView.PERSONAL_EVENTS) {
+    const isUserEvents = eventView === eventsView.PERSONAL_EVENTS;
+    if (isUserEvents) {
       getUsersEvents(date)
-        .then(({ data }) => onSuccess(data))
+        .then(({ data }) => onSuccess(data, isUserEvents))
```

Comment:
could just have true here instead of isUserEvents for readability - but its a question of personal taste so not too worried here